packages/dd-trace/src/opentracing/span_context.js

```diff
@@ -36,13 +36,15 @@ class DatadogSpanContext {
         ? this._trace.tags[TRACE_ID_128] + this._traceId.toString(16).padStart(16, '0')
         : this._traceId.toString(16).padStart(32, '0')
     }
+    if (this._traceId === undefined) return 'unavailable'
```

Comment:
Why not just handle this in the caller? (or even better not rely on the span context for this)

Also, this is not tested.

Reply:
can we rely on the user to do exception handling? They shouldn't call these two functions if they are only propagating baggage, but I'm not sure if I can trust them

Reply:
on a second thought you are probably right, I'll close this PR

Reply:
If they are using OTel they would never be using our internal `SpanContext`.